src/Microsoft.AspNetCore.Razor.Language/CompositeRazorProject.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public class CompositeRazorProject : RazorProject
```

Comment:
These should be internal and subclass `RazorProjectSystem`

Reply:
Should also live in Razor.Tools until we have a business need to bring it over to Razor.Language